integration-tests/src/test/resources/features/error-handling.feature

```diff
@@ -176,4 +176,58 @@ Feature: Error handling tests
     }
     """
     Then the polling error handler endpoint of the Bridge "pollingEhBridge" contains message "Polling error handler message from ${bridge.pollingEhBridge.id} bridge id" within 1 minute
-    And the polling error handler endpoint of the Bridge "pollingEhBridge" has only 1 error message within 1 minute
\ No newline at end of file
+    And the polling error handler endpoint of the Bridge "pollingEhBridge" has only 1 error message within 1 minute
+
+  @errorhandlingdlqheaders
+  Scenario: A failure occurs and a cloud event message is sent by managed connector to DLQ with correct headers
+    Given authenticate against Manager
+    And create a new Bridge with body:
+    """
+    {
+        "name": "ehBridge",
+        "cloud_provider": "aws",
+        "region": "us-east-1",
+        "error_handler": {
+            "type": "webhook_sink_0.1",
+            "parameters": {
+                "endpoint": "https://webhook.site/${env.webhook.site.uuid}"
```

Comment:
To be synced with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1208 , based on what will get merged first.